src/cohort/Cohort.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import './Cohort.css';
+import StudentLineup from '../student-list/StudentLineup';
+import GroupList from '../group-list/GroupList';
+
+class Cohort extends Component {
+  constructor(){
+    super();
+    this.state = {checkedStudents: [], groups: [], students: []};
+    this.toggleStudent = this.toggleStudent.bind(this);
+    this.submit = this.submit.bind(this);
+    this.delete = this.delete.bind(this);
+    this.addToList = this.addToList.bind(this);
+    this.post = this.post.bind(this);
+  }
+
+  post(){
+    console.log(JSON.stringify(this.state.groups))
+    fetch(`https://johariwindowapi.herokuapp.com/api/v1/assignments`, {
```

Comment:
We should do a sweep of the app tomorrow and look for console.logs. I will add a chore to the tracker.